### PR TITLE
Declarative setup, upgrades

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,34 @@
+[metadata]
+name = minique
+version: attr:minique.__version__
```
Comment (ruksi) [inline setup.cfg:3]: `version = attr:minique.__version__` ?

**Review by ruksi — CHANGES_REQUESTED** [review]
otherwise looks proper :tophat:

File: setup.cfg (hunk 1 above)
Comment (akx) [reply setup.cfg:3]: Good catch. Wonder how this works at all...

Comment (ruksi) [reply setup.cfg:3]: :shrug:

**Comment by akx** [comment]
Merging with @ruksi's single comment (excellent catch!) taken care of.

**Pull request merged**